catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -38,9 +41,11 @@
 
```

Comment:
✏️ In the description of the command, may be useful to add something like `By default, prints out all Metacard and Input transformers.`